flatlaf-core/src/main/resources/com/formdev/flatlaf/FlatLaf.properties

```diff
@@ -337,6 +337,7 @@ MenuBar.border=com.formdev.flatlaf.ui.FlatMenuBarBorder
 MenuBar.background=@menuBackground
 MenuBar.hoverBackground=@menuHoverBackground
 MenuBar.itemMargins=3,8,3,8
+MenuBar.underlineSelectionColor=$TabbedPane.underlineColor
```

Comment:
Here I don't know if it is correct.
Probably the best solution to not change the actual behavior is add a fallback to the value of MenuItem.underlineSelectionColor